### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by JP-1947** [rejetto/hfs#718] [Q&A]
Hi,

I am new to this. If I want to share a file with someone, do I need to keep the program open after I give them a link? Is there a way for them to access the link if the computer is off?
thank you

**Comment by PH5HP** [comment]
Hi,

The server must be running if he want to download the file using the link you gave him.

Kind regards,
Henk - https://ph5hp.nl - https://fileserver.ph5hp.nl
